code/game/objects/items/weapons/chaplain_items.dm

```diff
@@ -0,0 +1,110 @@
+/obj/item/weapon/nullrod
+	name = "null rod"
+	desc = "A rod of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullrod"
+	item_state = "nullrod"
+	slot_flags = SLOT_BELT
+	force = 15
+	throw_speed = 1
+	throw_range = 4
+	throwforce = 10
+	w_class = ITEMSIZE_SMALL
+	drop_sound = 'sound/items/drop/sword.ogg'
+	pickup_sound = 'sound/items/pickup/sword.ogg'
+	var/cooldown = 0 // floor tap cooldown
+
+	suicide_act(mob/user)
+		var/datum/gender/T = gender_datums[user.get_visible_gender()]
+		to_chat(viewers(user),"<span class='danger'>[user] is impaling [T.himself] with the [src.name]! It looks like [T.he] [T.is] trying to commit suicide.</span>")
+		return (BRUTELOSS|FIRELOSS)
+
+/obj/item/weapon/nullrod/staff
+	name = "null staff"
+	desc = "A staff of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullstaff"
+	item_state = "nullstaff"
+	slot_flags = SLOT_BELT | SLOT_BACK
+	w_class = ITEMSIZE_LARGE
+
+/obj/item/weapon/nullrod/orb
+	name = "null sphere"
+	desc = "An orb of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullorb"
+	item_state = "nullorb"
+
+/obj/item/weapon/nullrod/athame
+	name = "null athame"
+	desc = "An athame of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullathame"
+	item_state = "nullathame"
+
+/obj/item/weapon/nullrod/verb/change(mob/user)
+	set name = "Reconfigure Null Item"
+	set category = "Object"
+	set src in usr
+
+	var/list/options = list()
+	options["Null rod"] = list(/obj/item/weapon/nullrod)
+	options["Null staff"] = list(/obj/item/weapon/nullrod/staff)
+	options["Null sphere"] = list(/obj/item/weapon/nullrod/orb)
+	options["Null athame"] = list(/obj/item/weapon/nullrod/athame)
+	var/choice = input(user,"What form would you like your obsidian relic to take?") as null|anything in options
+	if(src && choice)
+		to_chat(user, SPAN_NOTICE("You start reconfiguring your obsidian relic."))
+		if(!do_after(user, 2 SECONDS))
+			return
+		var/list/things_to_spawn = options[choice]
+		for(var/new_type in things_to_spawn) // Spawn the null item
+			var/atom/movable/AM = new new_type(get_turf(src))
+			if(istype(AM, /obj/item/weapon/nullrod))
+				to_chat(user, "You have reconfigured your obsidian relic to \the [AM].")
+		qdel(src)
```

Comment:
You should `user.unEquip(src)` before you qdel it, and then `user.put_in_hands(AM)` to try and put the new item in the user's hands.